ims/cli/cli.py

```diff
@@ -576,6 +576,31 @@ def get_node_ip(project, node):
             click.echo(ret[constants.MESSAGE_KEY])
 
 
+@node.command('iscsi', help='Get iSCSI target info for a provisioned node')
+@click.argument(constants.PROJECT_PARAMETER)
+@click.argument(constants.NODE_NAME_PARAMETER)
+@bmi_exception_wrapper
+def get_iscsi_target(project, node):
```

Comment:
I think this should be in the iscsi sub command? You can call it info or some other apt name